staging/src/k8s.io/metrics/hack/update-codegen.sh

```diff
@@ -28,13 +28,13 @@ CODEGEN_PKG=${CODEGEN_PKG:-$(cd ${SCRIPT_ROOT}; ls -d -1 ./vendor/k8s.io/code-ge
 #
 # we skip informers and listers for metrics, because we don't quite support the requisite operations yet
 # we skip generating the internal clientset as it's not really needed
-${CODEGEN_PKG}/generate-internal-groups.sh deepcopy,conversion \
+"${CODEGEN_PKG}"/generate-internal-groups.sh deepcopy,conversion \
```

Comment:
quote the whole file path

Reply:
hmm do we have a style guide? agree with this, but it would be nice to be able to point to a set of rules (maybe the google one?)